### PR TITLE
Http endpoint reporting metrics

### PR DESCRIPTION
**Pull request by weralabaj**
This PR introduces and http endpoints reporting metrics gathered from different endpoints with a single raw JSON object in a following format:

```
{    
 "NServiceBus.Endpoints" : {
   "Endpoint1" : { 
     "Property1" : 1,
     "Property2" : "test1"
   },
   "Endpoint2" : { 
     "Property1" : 2,
     "Property2" : "test2"
   }
}
```

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
LGTM.

File: src/ServiceControl.Monitoring.AcceptanceTests/Tests/When_having_raw_data_provided.cs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    using global::Newtonsoft.Json;
+    using global::Newtonsoft.Json.Linq;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+    using ObjectBuilder;
+    using ServiceControl.Monitoring;
+    using ServiceControl.Monitoring.Http;
+
+    public class When_having_raw_data_provided : NServiceBusAcceptanceTest
```
Comment (weralabaj) [inline src/ServiceControl.Monitoring.AcceptanceTests/Tests/When_having_raw_data_provided.cs]: I find it hard to understand this test, but I think modifying the name slightly should solve the issue for me. e.g.

When_having_raw_data_provided  => Endpoint_using_metrics_provider
Should_report_via_http => Should_report_metrics_via_http

**Comment by weralabaj** [comment]
One little comment, LGTM

**Comment by Scooletz** [comment]
Merged after applying your changes @weralabaj

**Pull request merged**